opentutor_classifier/tests/test_train_classifier.py

```diff
@@ -101,31 +101,50 @@ def _test_train_and_predict(
         )
 
 
+@pytest.mark.only
 @pytest.mark.parametrize(
     "example,arch,confidence_threshold,expected_training_result,expected_accuracy",
     [
+        # (
```

Comment:
do we really not want these other tests any more? probably need to be uncommented?